packages/babel-preset-typescript/src/index.js

```diff
@@ -1,10 +1,10 @@
 import { declare } from "@babel/helper-plugin-utils";
 import transformTypeScript from "@babel/plugin-transform-typescript";
 
-export default declare(api => {
+export default declare((api, { jsxPragma = "React" }) => {
```

Comment:
This default is already set for the transform itself, so it would better not to repeat it again.